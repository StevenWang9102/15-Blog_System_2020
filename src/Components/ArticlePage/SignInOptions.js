import React from "react";
import { HashRouter as Router, Link } from "react-router-dom";

export const SignInOptions = props => {
  return (
    <Router>
    <div className='container page'>
      <div className='row'>
        <div className='col-md-12'>
          <Link to='sign_in'>
            <span> Sign in </span>
          </Link>
          or
          <Link to='sign_up'> Sign up </Link>
          to add comments on this article.
        </div>
      </div>
    </div>
    </Router>
  );
};
