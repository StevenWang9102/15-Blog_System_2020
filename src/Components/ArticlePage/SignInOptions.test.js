import React from "react";
import { SignInOptions } from "./SignInOptions";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <SignInOptions/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
