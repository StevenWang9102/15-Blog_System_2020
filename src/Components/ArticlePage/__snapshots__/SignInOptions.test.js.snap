// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container page"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <a
        href="#/sign_in"
        onClick={[Function]}
      >
        <span>
           Sign in 
        </span>
      </a>
      or
      <a
        href="#/sign_up"
        onClick={[Function]}
      >
         Sign up 
      </a>
      to add comments on this article.
    </div>
  </div>
</div>
`;
